src/inference_profiler.cc

```diff
@@ -561,6 +561,9 @@ InferenceProfiler::Profile(
   is_stable = false;
   meets_threshold = true;
 
+  RETURN_IF_ERROR(
+      dynamic_cast<ConcurrencyManager*>(manager_.get())
+          ->PerformWarmup(concurrent_request_count, warmup_request_count));
```

Comment:
How does stats and timestamps collected during the warmup get discarded? (I'm guessing this is achieved through `WaitForWarmupAndCleanup` by cleaning up all the worker threads that contain the record/data, but just want to confirm).

Reply:
Do we want to clean up the warmup data or should it be collected in a separate portion of the export json?
I could see a case where user's would want that data too.

Reply:
The stats are in `thread_stats_`, which gets cleared in `WaitForWarmupAndCleanup()`.

The timestamps only get saved when `ProfileHelper()` gets called, but warmup happens before that.

Reply:
But do we want to throw this data away?

Reply:
Yup. If someone wanted it they can skip using the warmup feature and just specify more requests via `--request-count`

Reply:
sounds good to me. will defer to @debermudez on closing this comment.

Reply:
yea thats really reasonable.
appreciate the answer.